strictEqual.js

```diff
@@ -0,0 +1,9 @@
+const strictEqual = (a, b) => {
```

Comment:
`a` and `b` aren't great names, even `val1` and `val2` are slightly better. Generally you never see single letter names outside of for loop iterators like `i`